fist/tests.c

```diff
@@ -389,6 +390,44 @@ static char *test_serialize_hmap() {
     return 0;
 }
 
+
+static char *test_create_bst() {
+    struct bst_node *root = bst_create("Hello", NULL);
+    mu_assert("bst_create: Equals 'Hello'", !strcmp(root->key, "Hello"));
+    bst_free(root);
+    return 0;
+}
+
+int test_helper_handler1() {
+    return 0;
+}
+char test_helper_handler2() {
+    return 0;
+}
+void test_helper_handler3() {
+
+}
+int* test_helper_handler4() {
+    return NULL;
+}
+static char *test_insert_and_search_bst(){
+    struct bst_node *root = bst_create("Hello World", NULL);
+    bst_insert(&root, "BSTn",test_helper_handler1);
```

Comment:
Now that bst stores `void *`, you could just pass in an int or string or something instead of making `test_helper_handler{1..4}`

Reply:
Clean up